webui/react/src/pages/ModelDetails/ModelHeader.tsx

```diff
@@ -148,6 +149,12 @@ const ModelHeader: React.FC<Props> = ({
     [deleteModelModal, modelEditModal, modelMoveModal, onSwitchArchive],
   );
 
+  const listPath = useMemo(
+    () =>
+      model.workspaceId ? paths.workspaceDetails(model.workspaceId, 'models') : paths.modelList(),
+    [model.workspaceId],
+  );
+
```

Comment:
nit: given that the model is already being passed into the modal component, and the path is only used when the modal is confirmed, can we move this into the deleteModelModal component?

Reply:
That makes sense; with the new commit I've changed the prop to `redirectOnDelete` and did the redirect path in the modal code